src/types.cc

```diff
@@ -111,7 +111,7 @@ Status unified_site_of_yaml(const YAML::Node& yaml, const vector<pair<string,siz
         const auto n_urange = (*p)["range"];
         VR(n_urange, "missing 'range' field in unification entry");
         S(range_of_yaml(n_urange, contigs, urange, ans.pos.rid));
-        VR(ans.pos.contains(urange), "unification entry range isn't contained within site range");
+        VR(ans.pos.overlaps(urange), "unification entry range does not overlap site range");
```

Comment:
I believe in the newer conception of unified_site, it will be possible for a range in the `unification` to be not contained within the unified_site range (e.g. when the input gvcf is more padded than the unified representation, for example the trim_input example, which will error on this current line). But I think it still holds true that they must at least overlap.